hivemq-edge/src/main/java/com/hivemq/api/utils/BridgeUtils.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2019-present HiveMQ GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hivemq.api.utils;
+
+import com.hivemq.bridge.config.BridgeTls;
+import com.hivemq.bridge.config.BridgeWebsocketConfig;
+import com.hivemq.bridge.config.CustomUserProperty;
+import com.hivemq.bridge.config.LocalSubscription;
+import com.hivemq.bridge.config.MqttBridge;
+import com.hivemq.bridge.config.RemoteSubscription;
+import com.hivemq.edge.api.model.Bridge;
+import com.hivemq.edge.api.model.BridgeCustomUserProperty;
+import com.hivemq.edge.api.model.BridgeSubscription;
+import com.hivemq.edge.api.model.LocalBridgeSubscription;
+import com.hivemq.edge.api.model.Status;
+import com.hivemq.edge.api.model.WebsocketConfiguration;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.stream.Collectors;
+
+public class BridgeUtils {
+
+    public static @NotNull Bridge convert(final @NotNull MqttBridge mqttBridge, final @NotNull Status status) {
```

Comment:
Would be nicer to have this directly on one of the classes as a to or from-method.

Reply:
not sure, we did this for data hub and it turned out to be a semi good idea. I think a 3rd component is actually correct, as the Bridge should not care about the MqttBridge and vice versa.